script/models/windowtypes.js

```diff
@@ -1,10 +1,18 @@
+/**
+ * @module bigscreenplayer/models/windowtypes
+ * @readonly
+ * @enum {string}
+ */
 define(
   'bigscreenplayer/models/windowtypes',
   function () {
     'use strict';
-    return {
+    return /** @alias module:bigscreenplayer/models/windowtypes */ {
+      /** Media with a duration */
       STATIC: 'staticWindow',
+      /** Media with a start without an initial duration */
```

Comment:
Reword as 'start time but without...' maybe? The sentence structure just confused me :)